cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -2742,3 +2742,46 @@ def test_organism_tissue_ontology_term_id__invalid(
         obs.loc[obs.index[0], "organism_tissue_ontology_term_id"] = organism_tissue_ontology_term_id
         validator.validate_adata()
         assert validator.errors == [error + " " + zebrafish_error_message_suffix]
+
+
+class TestFruitFly:
+    """
+    Tests for the fruit fly schema
+    """
+
+    def test_development_stage_ontology_term_id_fruitfly(self, validator_with_adata):
```

Comment:
let's add a test that 'unknown' also passes validation

Reply:
updated